formatTest/unit_tests/expected_output/modules.re

```diff
@@ -445,9 +447,8 @@ let module Id (X: Type) => X;
 let module Compose
            (F: Type => Type)
            (G: Type => Type)
-           (X: Type) => F (
-  G X
-);
+           (X: Type) =>
+  F (G X);
```

Comment:
I agree - they should be consistent. Right now, they are consistent, correct? Were you suggesting that whatever we do with function args, we do with functor args?